### DISCUSSION TITLE
Could we model reforms as branches?

### DISCUSSION THREAD

**Discussion by MaxGhenis** [PolicyEngine/policyengine-us#1264] [Ideas]
This would be the most gitty approach. Could we have reform OpenFisca US branches make their way into PolicyEngine somehow?

What made me think of this is the Inflation Reduction Act, which will be signed into law tomorrow. It'd have been slick to just have it in a branch that we then merge upon signing.

Even cooler would be to make the branches from bots associated with legislators or organizations.

**Comment by nikhilwoodruff** [comment]
Yeah! We could just put it in a draft PR right?

**Reply by MaxGhenis** [reply]
Yes, the hard part I think will be loading different branches in PolicyEngine
